ghost/core/test/e2e-server/services/milestones.test.js

```diff
@@ -11,6 +11,7 @@ const milestonesService = require('../../../core/server/services/milestones');
 let agent;
 let counter = 0;
 let membersCounter = 0;
+const inFifteenDays = 1000 * 60 * 60 * 24 * 15;
```

Comment:
I don't really care, but we would generally call this `fifteenDays` without the `'in'` prefix

Reply:
Moved test updates to https://github.com/TryGhost/Ghost/pull/16314